src/Microsoft.ML.Data/Prediction/CalibratorCatalog.cs

```diff
@@ -0,0 +1,438 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.ML.Core.Data;
+using Microsoft.ML.Calibrator;
+using Microsoft.ML.Runtime;
+using Microsoft.ML.Runtime.Data;
+using Microsoft.ML.Runtime.Internal.Calibration;
+using Microsoft.ML.Runtime.Model;
+using Microsoft.ML.Runtime.Training;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.ML.Data;
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<PlattCalibrator>), typeof(PlattCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", PlattCalibratorTransformer.LoadName)]
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<NaiveCalibrator>), typeof(NaiveCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", NaiveCalibratorTransformer.LoadName)]
+
+[assembly: LoadableClass(typeof(CalibratorTransformer<PavCalibrator>), typeof(PavCalibratorTransformer), null,
+    typeof(SignatureLoadModel), "", PavCalibratorTransformer.LoadName)]
+
+namespace Microsoft.ML.Calibrator
+{
+
+    /// <summary>
+    /// Base class for CalibratorEstimators. CalibratorEstimators take a the result of a <see cref="BinaryClassifierScorer"/>, so an <see cref="IDataView"/>
+    /// that contains a "Score" column, and converts the scores to probabilities(through binning, interpolation etc.), based on the <typeparamref name="TICalibrator"/> type.
```

Comment:
`CalibratorEstimators take a the result of a <see cref="BinaryClassifierScorer"/>, so an <see cref="IDataView"/> that contains a "Score" column,` --->`CalibratorEstimators take an <see cref="IDataView"/> (the output of a <see cref="BinaryClassifierScorer"/>) that contains a "Score" column`

Reply:
thanks!

---
In reply to: [240332534](https://github.com/dotnet/machinelearning/pull/1816#discussion_r240332534) [](ancestors = 240332534)